src/main/webapp/app/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -96,4 +102,16 @@ export class ListOfComplaintsComponent implements OnInit {
     back() {
         this.location.back();
     }
+
+    callback() {}
+
+    triggerAddressedComplaints() {
+        this.showAddressedComplaints = !this.showAddressedComplaints;
+
+        if (this.showAddressedComplaints) {
+            this.complaintsToShow = this.complaints;
+        } else {
+            this.complaintsToShow = this.complaints.filter(c => c.accepted === undefined);
```

Comment:
At first I thought that if the checkbox is activate ALSO the addressed commands are shown.
But after reading this again, this is more of a toggle.
I think it would be more clear if the it was a radio button or a toggle button (https://ng-bootstrap.github.io/#/components/buttons/examples) - show addressed vs. show unaddressed.

Reply:
I was wrong about this - never mind.